internal/handler/msg_createuser.go

```diff
@@ -108,65 +103,39 @@ func (h *Handler) MsgCreateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 
 	common.Ignored(document, h.L, "writeConcern", "authenticationRestrictions", "comment")
 
-	defMechanisms := must.NotFail(types.NewArray("SCRAM-SHA-1", "SCRAM-SHA-256"))
+	allMechanisms := must.NotFail(types.NewArray("SCRAM-SHA-1", "SCRAM-SHA-256"))
 
-	mechanisms, err := common.GetOptionalParam(document, "mechanisms", defMechanisms)
+	mechanisms, err := common.GetOptionalParam(document, "mechanisms", allMechanisms)
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
-	var credentials *types.Document
+	if mechanisms.Len() == 0 {
+		return nil, handlererrors.NewCommandErrorMsg(
```

Comment:
Is there new test to test this case?

Reply:
There is https://github.com/FerretDB/FerretDB/blob/main/integration/users/create_user_test.go#L107

Reply:
Great!